tests/foreman/cli/test_leapp_client.py

```diff
@@ -164,8 +164,11 @@ def verify_target_repo_on_satellite(
     )
     repo_names = [out['name'] for out in cmd_out]
     if target_rhel_major_ver == '9':
-        assert RHEL_REPOS['rhel9_bos']['name'] in repo_names
-        assert RHEL_REPOS['rhel9_aps']['name'] in repo_names
+        # This is workaround until leapp doesn't support RHEL 9.4 for upgrade.
+        assert 'BaseOS RPMs 9.3' in repo_names
+        assert 'AppStream RPMs 9.3' in repo_names
+        # assert RHEL_REPOS['rhel9_bos']['name'] in repo_names
+        # assert RHEL_REPOS['rhel9_aps']['name'] in repo_names
```

Comment:
- Why we are not using existing assert statement instead of hardcoded strings?
- If we want to use hardcoded string then please remove the commented statements (line 170 & 171) and use globaly declared varibale `RHEL9_VER` like below
```
assert f'BaseOS RPMs {RHEL9_VER}' in repo_names
assert f'AppStream RPMs {RHEL9_VER}' in repo_names
```

Reply:
Leapp doesn't support latest rhel versions yet so RHEL_REPOS['rhel9_bos']['name'] will fetch the latest rhel version(9.4) and assertion fails there.